app/routes/admin/tenants.py

```diff
@@ -58,12 +58,58 @@
 )
 
 
-@router.post("", response_model=CreateTenantResponse)
+@router.post("", response_model=CreateTenantResponse, summary="Create a new Tenant")
 async def create_tenant(
     body: CreateTenantRequest,
     admin_auth: AcaPyAuthVerified = Depends(acapy_auth_tenant_admin),
 ) -> CreateTenantResponse:
-    """Create a new tenant."""
+    """
+    Create a new tenant
+    ---
+
+    Use this endpoint to create a new tenant.
+    This will create a new wallet (the tenant) and onboard it with the requested roles.
+
+    The `wallet_name` is an optional field that allows you to assign a name to the wallet,
+    useful with `get_tenants` to fetch wallets by wallet name.
+
+    `image_url` is an optional field that allows tenant-admin to assign an image to the wallet.
+
+    `group_id` is an optional field that allows the tenant-admin to assign the wallet to a group.
```

Comment:
Note: the admin can't control group_id. The group_id **query param** has `include_in_schema=False`, but it does still appear in request bodies... hmm, so we perhaps we should mark it as hidden in the request body too. AFAIK from Robbie's work we inject group_id to request bodies, and will override whatever the client enters there. So, it shouldn't be in the model or docstrings imo